analysis_options.yaml

```diff
@@ -1,13 +1,211 @@
-include: package:effective_dart/analysis_options.1.3.0.yaml
```

Comment:
Is there any package that we can use instead that includes most of these rules? Maybe the vgv one if flame_lint doesn't cut it?

Reply:
i copied from the vgv one with some rules changed

Reply:
Why not use the VGV one and just explicitly change the rules that you want to change?

Reply:
ah okay